GDAS-validation/gdas_config/config.anal

```diff
@@ -0,0 +1,155 @@
+  export DIAG_TARBALL="YES"
+fi
+
+export npe_gsi=${npe_anal}
+
+if [[ "${CDUMP}" == "gfs" ]] ; then
+  export npe_gsi=${npe_anal_gfs}
+  export nth_anal=${nth_anal_gfs}
+fi
+
+# Set parameters specific to L127
+if [[ ${LEVS} = "128" ]]; then
+  export GRIDOPTS="nlayers(63)=1,nlayers(64)=1,"
+  export SETUP="gpstop=55,nsig_ext=45,${SETUP:-}"
+fi
+
+# Set namelist option for LETKF
+export lobsdiag_forenkf=".false."  # anal does not need to write out jacobians
+                   # set to .true. in config.eobs and config.eupd
+
+# Do not process the following datasets
+export GSNDBF=${GSNDBF:-/dev/null}
+export AMSREBF=${AMSREBF:-/dev/null}
+export SSMITBF=${SSMITBF:-/dev/null}
+export AMSR2BF=${AMSR2BF:-/dev/null}
+
+
+# Set default values for info files and observation error
+#  NOTE:  Remember to set PRVT in config.prep as OBERROR is set below
+export CONVINFO=${FIXgsi}/global_convinfo.txt
+export OZINFO=${FIXgsi}/global_ozinfo.txt
+export SATINFO=${FIXgsi}/global_satinfo.txt
+export OBERROR=${FIXgsi}/prepobs_errtable.global
+
+
+# Use experimental dumps in EMC GFS v16 parallels
+if [[ ${RUN_ENVIR} == "emc" ]]; then
+  #   Set info files and prepobs.errtable.global for GFS v16 retrospective parallels
+  if [[ "${PDY}${cyc}" -ge "2019021900" && "${PDY}${cyc}" -lt "2019110706" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2019021900
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2019021900
+  fi
+
+  #   Place GOES-15 AMVs in monitor, assimilate GOES-17 AMVs, assimilate KOMPSAT-5 gps
+  if [[ "${PDY}${cyc}" -ge "2019110706" && "${PDY}${cyc}" -lt "2020040718" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2019110706
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2019110706
+  fi
+
+  #   Assimilate 135 (T) & 235 (uv) Canadian AMDAR observations
+  if [[ "${PDY}${cyc}" -ge "2020040718" && "${PDY}${cyc}" -lt "2020052612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020040718
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2020040718
+  fi
+
+  #   Assimilate COSMIC-2
+  if [[ "${PDY}${cyc}" -ge "2020052612" && "${PDY}${cyc}" -lt "2020082412" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020052612
+    export OBERROR=${FIXgsi}/gfsv16_historical/prepobs_errtable.global.2020040718
+  fi
+
+  #   Assimilate HDOB
+  if [[ "${PDY}${cyc}" -ge "2020082412" && "${PDY}${cyc}" -lt "2020091612" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020082412
+  fi
+
+  #   Assimilate Metop-C GNSSRO
+  if [[ "${PDY}${cyc}" -ge "2020091612" && "${PDY}${cyc}" -lt "2021031712" ]]; then
+    export CONVINFO=${FIXgsi}/gfsv16_historical/global_convinfo.txt.2020091612
+  fi
+
+  #   Assimilate DO-2 GeoOptics
+  if [[ "${PDY}${cyc}" -ge "2021031712" && "${PDY}${cyc}" -lt "2021091612" ]]; then
```

Comment:
This logic changes the `convinfo` file used by 2021080100 gdasanal.  A check of the log file shows

```
+++ config.anal[48]: export CONVINFO=/work2/noaa/da/rtreadon/gdas-validation/global-workflow/fix/gsi/global_convinfo.txt
+++ config.anal[48]: CONVINFO=/work2/noaa/da/rtreadon/gdas-validation/global-workflow/fix/gsi/global_convinfo.txt



+++ config.anal[91]: [[ 2021080100 -lt 2021091612 ]]
+++ config.anal[92]: export CONVINFO=/work2/noaa/da/rtreadon/gdas-validation/global-workflow/fix/gsi/gfsv16_historical/global_convinfo.txt.2021031712
+++ config.anal[92]: CONVINFO=/work2/noaa/da/rtreadon/gdas-validation/global-workflow/fix/gsi/gfsv16_historical/global_convinfo.txt.2021031712



+ exglobal_atmos_analysis.sh[299]: CONVINFO=/work2/noaa/da/rtreadon/gdas-validation/global-workflow/fix/gsi/gfsv16_historical/global_convinfo.txt.2021031712
```

Reply:
FYI I just grabbed these from develop of global-workflow. Is this not the intended behavior (to use the historical file), or @RussTreadon-NOAA are you just noting that this is using a historical file?

Reply:
I agree.  We don't want to use the historical files.  I added a comment below that we can reset `CONVINFO` and `SATINFO` at the end of `config.anal` to use the intended files.   

FYI, I installed this PR on Orion to step through the clone, build, and run process to mimic a user's experience.  It's an enlightening process.

Reply:
Addressed via 37fea66.